app/models/animal.rb

```diff
@@ -19,7 +19,8 @@ class Animal < ActiveRecord::Base
 
   scope :searchable, -> { where(searchable: true) }
 
-  VALID_INTERESTS = %w(keep sell trade loan).freeze
+  VALID_INTERESTS = %w(keep sell trade loan sell\ or\ trade\ in\ 1-2\ years looking\ for\ mate).freeze
```

Comment:
I'd opt for: `VALID_INTERESTS = ['keep', 'sell', 'trade', 'loan', 'sell or trade in 1-2 years', 'looking for mate'].freeze` in this case. More readable. But :+1: of course.